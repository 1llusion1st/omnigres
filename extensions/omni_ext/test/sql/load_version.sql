SELECT omni_ext.load('omni_ext_test', 'unknown');
SELECT omni_ext.load('omni_ext_test', '0.1');