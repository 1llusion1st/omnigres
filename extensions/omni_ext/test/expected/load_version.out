SELECT omni_ext.load('omni_ext_test', 'unknown');
 load 
------
 
(1 row)

SELECT omni_ext.load('omni_ext_test', '0.1');
 load 
------
 0.1
(1 row)

